docs/node-operators/how-to/solve-frequent-issues.md

```diff
@@ -255,4 +265,12 @@ vega data node stopped with error: failed to flush subscriber:flushing ledger: f
 
 1. Update your PostgreSQL config file. This is usually the config is located at `cat /etc/postgresql/14/main/postgresql.conf`). Change the max_locks_per_transaction to a bigger number, such as 256 or higher.
 2. Restart your PostgreSQL server
-3. Start your data node.
\ No newline at end of file
+3. Start your data node.
+
+## Problem: You encounter deadlocking issues with the Postgres database causing it to crash.
+
+We have found during testing that for some scenarios, such as snapshot restores, Postgres can encounter deadlocks when refreshing the continuous aggregates. This leads to Postgres panics and causes the data node to crash.
+
+### Solution: Ensure you have the noatime flag set for your mounted drives.
+
+Ubuntu defaults do not set the noatime flag for mounted drives, this can have a significant performance impact for the database and causes the deadlocking issues. While not a panacea for all deadlocking issues, it is a good first step to take.
```

Comment:
```suggestion
Ubuntu defaults do not set the `noatime` flag for mounted drives, this can have a significant performance impact for the database and causes the deadlocking issues. While not a panacea for all deadlocking issues, it is a good first step to take.
```